build.sbt

```diff
@@ -662,6 +664,7 @@ lazy val metalsDependencies = project
       "ch.epfl.scala" % "bloop-maven-plugin" % V.mavenBloop,
       "ch.epfl.scala" %% "gradle-bloop" % V.gradleBloop,
       "com.sourcegraph" % "semanticdb-java" % V.javaSemanticdb,
+      "ch.epfl.scala" %% "scala-debug-adapter" % V.debugAdapter intransitive (),
```

Comment:
Used `intransitive` to avoid the version conflict in transitive dependencies (`scala-parallel-collections`).